### DISCUSSION TITLE
[FAQ] Why does Velero/restic not support hostPath volumes?

### DISCUSSION THREAD

**Discussion by dsu-igeek** [vmware-tanzu/velero#3378] [Community support Q&A] [answered]
Why does Velero/restic not support hostPath volumes?

**Comment by dsu-igeek** [comment, marked as answer]
Velero’s restic daemonset uses a hostPath mount of `/var/lib/kubelet/pods` to get access to pods’ volume mounts. If a pod is using a hostPath mount, it won’t be mounted under that directory, so Velero isn’t able to access it.

**Comment by ghost** [comment]
You actually can set it based on [this](https://github.com/vmware-tanzu/helm-charts/blob/velero-2.32.0/charts/velero/values.yaml#L392-L393)

This worked for me (I have nodefs living in `/data/kubelet` instead of `/var/lib/kubelet`):

```
restic:
  podVolumePath: /data/kubelet/pods
```

(This is the helm-chart based Velero installation).
